tests/backup/backup_share_test.go

```diff
@@ -4215,3 +4215,255 @@ var _ = Describe("{SwapShareBackup}", func() {
+						userBackupLocationMap[user] = map[string]string{userBackupLocationUID: userBackupLocationName}
+					}
+				})
+				Step(fmt.Sprintf("Create source and destination cluster from the user %s", user), func() {
+					log.InfoD(fmt.Sprintf("Creating source and destination cluster from the user %s", user))
+					nonAdminCtx, err := backup.GetNonAdminCtx(user, commonPassword)
+					log.FailOnError(err, "failed to fetch user %s ctx", user)
+					err = CreateApplicationClusters(orgID, "", "", nonAdminCtx)
+					log.FailOnError(err, "failed create source and destination cluster from the user %s", user)
+					clusterStatus, err := Inst().Backup.GetClusterStatus(orgID, SourceClusterName, nonAdminCtx)
+					log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
+					dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", SourceClusterName))
+					userClusterMap[user] = make(map[string]string)
+					for _, clusterName := range []string{SourceClusterName, destinationClusterName} {
+						userClusterUID, err := Inst().Backup.GetClusterUID(nonAdminCtx, orgID, clusterName)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching [%s] cluster uid", clusterName))
+						userClusterMap[user][clusterName] = userClusterUID
+					}
+				})
+				Step(fmt.Sprintf("Take backup of applications from the user %s", user), func() {
+					log.InfoD(fmt.Sprintf("Taking backup of applications from the user %s", user))
+					nonAdminCtx, err := backup.GetNonAdminCtx(user, commonPassword)
+					log.FailOnError(err, "failed to fetch user %s ctx", user)
+					var wg sync.WaitGroup
+					var mu sync.RWMutex
+					userBackupMap[user] = make(map[string]string)
+					createBackup := func(backupName string, namespace string) {
+						defer GinkgoRecover()
+						defer wg.Done()
+						appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, []string{namespace})
+						for backupLocationUID, backupLocationName := range userBackupLocationMap[user] {
+							err := CreateBackupWithValidation(nonAdminCtx, backupName, SourceClusterName, backupLocationName, backupLocationUID, appContextsToBackup, make(map[string]string), orgID, userClusterMap[user][SourceClusterName], "", "", "", "")
+							dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation and validation of backup [%s] of namespace (scheduled Context) [%s]", backupName, namespace))
+							break
+						}
+						mu.Lock()
+						defer mu.Unlock()
+						userBackupMap[user][backupName] = namespace
+					}
+					for _, namespace := range appNamespaces {
+						for i := 0; i < numberOfBackups; i++ {
+							backupName := fmt.Sprintf("%s-%s-%d-%v", BackupNamePrefix, namespace, i, time.Now().Unix())
+							wg.Add(1)
+							go createBackup(backupName, namespace)
+						}
+					}
+					wg.Wait()
+					log.Infof("The list of user backups taken are: %v", userBackupMap)
+				})
+			}
```

Comment:
I might be missing a step to verify backup object created by user is listed from admin pages. @ak-px